package.json

```diff
@@ -37,7 +37,8 @@
   },
   "homepage": "https://github.com/kriskuiper/cmd-digital-playground#readme",
   "dependencies": {
-    "dotenv-safe": "^8.2.0"
+    "dotenv-safe": "^8.2.0",
+    "slugify": "^1.4.0"
```

Comment:
Since it's only used prebuild you could install this as a dev-dependency. We also use dotenv-safe in serverless functions (if we need to) so that's why it is a normal dependency.